templates/docs/patterns/linked-logo-section/index.md

```diff
@@ -0,0 +1,161 @@
+
+## Full width
+
+<div class="embedded-example"><a href="/docs/examples/patterns/linked-logo-section/default" class="js-example" data-lang="jinja">
+View example of the linked logo section full-width pattern
+</a></div>
+
+## 50/50
+
+This variant can be used for adding a header and the logo section in different columns. This should only be used when there’s a maximum of 6 logos to avoid occupying too much vertical space.
+
+<div class="embedded-example"><a href="/docs/examples/patterns/linked-logo-section/50-50" class="js-example" data-lang="jinja">
+View example of the linked logo section 50-50 pattern
+</a></div>
+
+## 25/75
+
+This variant can be used for adding a header and the logo section in different columns, but can accommodate up to 9 logos. Be aware that the heading only occupies 25% width here, so only very short titles would be appropriate here
+
+<div class="embedded-example"><a href="/docs/examples/patterns/linked-logo-section/25-75" class="js-example" data-lang="jinja">
+View example of the linked logo section 25-75 pattern
+</a></div>
+
+## Jinja Macro
+
+The `vf_linked_logo_section` Jinja macro can be used to generate a linked logo list pattern. The API for the macro is shown below.
+
+### Parameters
+
+<div style="overflow: auto;">
+  <table>
+    <thead>
+      <tr>
+        <th style="width: 220px;">Name</th>
+        <th style="width: 160px;">Required?</th>
+        <th style="width: 160px;">Type</th>
+        <th style="width: 160px;">Default</th>
+        <th style="width: 250px;">Description</th>
+      </tr>
+      <tbody>
+        <tr>
+          <td>
+            <code>title_text</code>
+          </td>
+          <td>
+            Yes
+          </td>
+          <td>
+            String
+          </td>
+          <td>
+            N/A
+          </td>
+          <td>
+            The text to be displayed as the heading (<code>h2</code>).
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <code>layout</code>
+          </td>
+          <td>
+            No
+          </td>
+          <td>
+            One of:<br>
+            <code>'50/50'</code>,<br>
+            <code>'25/75'</code>,<br>
+            <code>'full-width'</code>
+          </td>
+          <td>
+            <code>full-wdith</code>
+          </td>
+          <td>
+            The intented grid layout for the section.
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <code>links</code>
+          </td>
+          <td>
+            Yes
+          </td>
+          <td>
+            Array
+          </td>
+          <td>
+            N/A
+          </td>
+          <td>
+            Array of image links, each including <code>href</code>, <code>label</code>, <code>text</code>, and <code>image_html</code> fields.
```

Comment:
I would also align this with the [pricing block](https://staging.vanillaframework.io/docs/patterns/pricing-block#jinja-macro) so we have a row for each image attribute